pkg/kubelet/apis/podresources/v1alpha1/api.pb.go

```diff
@@ -17,36 +17,23 @@ limitations under the License.
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
 // source: api.proto
 
-/*
```

Comment:
this looks spurious, I wouldn't expect this file to have changed

Reply:
Let me try reverting these changes and seeing if verification still passes

Reply:
I tried reverting changes to the file but 'make update' re-applied them.

Reply:
this still is unrelated to this change. if you drop it, does CI complain?

Reply:
Looks like CI is complaining if i do not include these changes

https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/84882/pull-kubernetes-bazel-test/1192910588246757380

compilepkg: missing strict dependencies:
	/b/f/w/pkg/kubelet/apis/podresources/v1alpha1/api.pb.go: import of "golang.org/x/net/context"
No dependencies were provided.
Check that imports in Go sources match importpath attributes in deps.
ERROR: /home/prow/go/src/github.com/kubernetes/kubernetes/cmd/genkubedocs/BUILD:52:1 GoCompilePkg pkg/kubelet/apis/podresources/v1alpha1/linux_amd64_race_stripped/go_default_library%/k8s.io/kubernetes/pkg/kubelet/apis/podresources/v1alpha1.a failed (Exit 1): builder failed: error executing command 

Should i keep the changes in the PR?

Reply:
Nvm i forgot to remove the updates to BUILD. After doing that CI looks happy